pkgs/by-name/op/openfortivpn-webview-qt/package.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, fetchFromGitHub, qt6, }:
+stdenv.mkDerivation rec {
+  pname = "openfortivpn-webview-qt";
+  version = "v1.1.2-electron";
+
+  src = fetchFromGitHub
+    {
+      owner = "gm-vm";
+      repo = "openfortivpn-webview";
+      rev = version;
+      hash = "sha256-BNotbb2pL7McBm0SQwcgEvjgS2GId4HVaxWUz/ODs6w=";
+    };
+  sourceRoot = "source/openfortivpn-webview-qt";
+
+
+  nativeBuildInputs = [ qt6.wrapQtAppsHook qt6.qtwebengine qt6.qmake ];
```

Comment:
I'd be surprised if `qtwebengine` goes here, see <https://nixos.org/manual/nixpkgs/unstable/#ssec-stdenv-dependencies>

Reply:
you are right.
I got confused because the build failed when it was not present as neither a `nativeBuildInput` or as a `buildInput`. But adding it as a `buildInput` is sufficient for it to build and execute correctly.